gsgmail_process_email.py

```diff
@@ -1094,4 +1107,18 @@ def _create_dict_hash(self, input_dict):
             self._base_connector.debug_print('Exception: ', e)
             return None
 
-        return hashlib.md5(input_dict_str.encode('utf-8')).hexdigest()
+        fips_enabled = self._get_fips_enabled()
+        # if fips is not enabled, we should continue with our existing md5 usage for generating hashes
+        # to not impact existing customers
+        try:
```

Comment:
what's the goal with this try/except? aren't they doing the same thing since the default encoding for `str.encode` is `utf-8`? (https://docs.python.org/3/library/stdtypes.html#str.encode)

also I think the except would currently raise an exception because dict_hash isn't defined in that block

Reply:
Great catch! I agree with your point. I've just remove the try-catch block as well as remove the "utf-8".